frinx-uniconfig/user-guide/uniconfig-operations/uniconfig-node-manager/rpc_checked-commit/readme.md

```diff
@@ -39,235 +39,147 @@ beginning of the transaction.
-                },
-                {
-                    "node-id": "IOSXR",
-                    "configuration-status": "complete"
-                }
-            ]
-        }
     }
 }'
 ```
 
 ```json RPC Response, Status: 200
-{
-    "output": {
-        "overall-status": "complete"
-    }
-}
 ```
 
 ### Failed Example
 
-RPC checked-commit input has 2 target nodes. One of them failed on
-configuration (IOSXRN). The output describes the result of the
-checked-commit.
+Nodes 'R1' and 'R2' has been changed. Node 'R1' has failed due to
+improper configuration. The output describes the result of the checked-commit.
 
 ```bash RPC Request
 curl --location --request POST 'http://localhost:8181/rests/operations/uniconfig-manager:checked-commit' \
 --header 'Accept: application/json' \
 --header 'Content-Type: application/json' \
 --data-raw '{
     "input": {
-        "target-nodes": {
-            "node": ["IOSXR","IOSXRN"]
-        }
     }
 }'
 ```
 
-```json RPC Response, Status: 200
+```json RPC Response, Status: 500
 {
-    "output": {
-        "overall-status": "fail",
-        "node-results": {
-            "node-result": [
-                {
-                    "node-id": "IOSXRN",
-                    "error-type": "uniconfig-error",
-                    "rollback-status": "complete",
-                    "error-message": "Supplied value \"GigabitEthernet0/0/0/1ghjfhjfhjfghj\" does not match required pattern \"^(([a-zA-Z0-9_]*\\d+/){3,4}\\d+)|(([a-zA-Z0-9_]*\\d+/){3,4}\\d+\\.\\d+)|(([a-zA-Z0-9_]*\\d+/){2}([a-zA-Z0-9_]*\\d+))|(([a-zA-Z0-9_]*\\d+/){2}([a-zA-Z0-9_]+))|([a-zA-Z0-9_-]*\\d+)|([a-zA-Z0-9_-]*\\d+\\.\\d+)|(mpls)|(dwdm)$\"\n",
-                    "configuration-status": "fail"
-                },
-                {
-                    "node-id": "IOSXR",
-                    "configuration-status": "complete",
-                    "rollback-status": "complete"
-                }
-            ]
+  "errors": {
+    "error": [
+      {
+        "error-type": "application",
+        "error-tag": "operation-failed",
+        "error-message": "Supplied value \"GigabitEthernet0/0/0/1ghjfhjfhjfghj\" does not match required pattern \"^(([a-zA-Z0-9_]*\\d+/){3,4}\\d+)|(([a-zA-Z0-9_]*\\d+/){3,4}\\d+\\.\\d+)|(([a-zA-Z0-9_]*\\d+/){2}([a-zA-Z0-9_]*\\d+))|(([a-zA-Z0-9_]*\\d+/){2}([a-zA-Z0-9_]+))|([a-zA-Z0-9_-]*\\d+)|([a-zA-Z0-9_-]*\\d+\\.\\d+)|(mpls)|(dwdm)$\"\n",
+        "error-info": {
+          "node-id": "R1",
+          "configuration-status": "fail",
+          "rollback-status": "complete"
         }
-    }
+      }
+    ]
+  }
 }
 ```
 
 ### Failed Example
 
-The RPC checked-commit input has 2 target nodes. One of them has failed
-on the changed fingerprint (IOSXRN) and the other has a bad
-configuration (IOSXR). The output describes the result of the
-checked-commit.
+Nodes 'R1' and 'R2' has been changed. Node 'R1' has failed on the
+changed fingerprint and the other has a bad configuration 'R2'.
```

Comment:
why 'R2' is not present in the error output?

Reply:
Done. Removed R2 from use case, because validation of fingerprint is processed before validation of configuration.